src/meshweb/views/join_record_viewer.py

```diff
@@ -0,0 +1,43 @@
+import logging
+from datetime import datetime, timezone
+
+from botocore.exceptions import ClientError
+from django.contrib.admin.views.decorators import staff_member_required
+from django.http import HttpRequest, HttpResponse
+from django.template import loader
+
+from meshapi.management.commands import replay_join_records
+from meshapi.util.join_records import JoinRecordProcessor
+
+
+@staff_member_required
```

Comment:
Super?

Reply:
Good catch. That was not my intention, but on further thought, this is just a viewer; I think it's valid for staff members (olvier, Matthew, etc) to be able to _view_ join records that need replaying.

Reply:
Sounds good for view.